dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10253,6 +10267,7 @@ int DynamicsFitter::estimateUnmeasuredExternalTorques(
         if (init->probablyMissingGRF.size() > trial)
         {
           init->probablyMissingGRF[trial][t] = true;
+          init->missingGRFReason[trial][t] = MissingGRFReason::unmeasuredTorque;
```

Comment:
Maybe for consistency with `forceDiscrepancy`, we call this `torqueDiscrepancy`?